gnomad/utils/vcf.py

```diff
@@ -493,6 +493,31 @@ def make_combo_header_text(
     return header_text
 
 
+def create_label_groups(
+    pops: List[str],
+    sexes: List[str],
+    groups: List[str] = GROUPS,
+    pop_sex_groups: List[str] = ["adj"],
+) -> List[Dict[str, List[str]]]:
+    """
+    Generate a list of label group dictionaries needed to populate info dictionary.
+
+    Label dictionaries are passed as input to `make_info_dict`.
+
+    :param pops: List of population names.
+    :param sexes: List of sample sexes.
+    :param groups: List of data types (raw, adj). Default is ["raw", "adj"].
```

Comment:
```suggestion
    :param groups: List of data types (raw, adj). Default is `GROUPS`, which is ["raw", "adj"].
```

Reply:
I'm also wondering if there is a better name for this now that you added `pop_sex_groups`. maybe `all_groups`?